apps/cowswap-frontend/src/modules/limitOrders/containers/TradeButtons/index.tsx

```diff
@@ -29,6 +33,8 @@ export function TradeButtons() {
 
   const tradeFormButtonContext = useTradeFormButtonContext(defaultText, confirmTrade)
 
+  const isDisabled = !warningsAccepted || !isTradeContextReady
```

Comment:
I guess, this is a key of the problem.
User could click "Review limit order" in a moment when `tradeContext` is null